packages/fluent/scss/card/_variables.scss

```diff
@@ -2,176 +2,180 @@
-/// Border color of the card header.
+/// The border color of the Card header.
 /// @group card
 $kendo-card-header-border: inherit !default; // header-border
 
-/// Horizontal padding of the card body.
+/// The horizontal padding of the Card body.
 /// @group card
 $kendo-card-body-padding-x: map.get( $kendo-spacing, 3 ) !default;
-/// Vertical padding of the card body.
+/// The vertical padding of the Card body.
 /// @group card
 $kendo-card-body-padding-y: map.get( $kendo-spacing, 3 ) !default;
 
-/// Horizontal padding of the card footer.
+/// The horizontal padding of the Card footer.
 /// @group card
 $kendo-card-footer-padding-x: $kendo-card-padding-x !default;
-/// Vertical padding of the card footer.
+/// The vertical padding of the Card footer.
 /// @group card
 $kendo-card-footer-padding-y: $kendo-card-padding-y !default;
-/// Top border width of the card footer.
+/// The top border width of the Card footer.
 /// @group card
 $kendo-card-footer-border-width: 0 !default;
-/// Background color of the card footer.
+/// The background color of the Card footer.
 /// @group card
 $kendo-card-footer-bg: inherit !default;
-/// Text color of the card footer.
+/// The text color of the Card footer.
 /// @group card
 $kendo-card-footer-text: inherit !default; // $kendo-component-header-text
-/// Border color of the card footer.
+/// The border color of the Card footer.
 /// @group card
 $kendo-card-footer-border: inherit !default; // $kendo-component-header-border
 
-/// Text size of the card title.
+/// The font size of the Card title.
 /// @group card
 $kendo-card-title-font-size: var( --kendo-font-size-lg, inherit ) !default;
-/// Font family of the card title.
+/// The font family of the Card title.
 /// @group card
 $kendo-card-title-font-family: var( --kendo-font-family, inherit ) !default;
-/// Text weight of the card title.
+/// The font weight of the Card title.
 /// @group card
 $kendo-card-title-font-weight: $kendo-card-header-font-weight !default;
-/// Line height of the card title.
+/// The line height of the Card title.
 /// @group card
 $kendo-card-title-line-height: var( --kendo-line-height, normal ) !default;
-/// Bottom margin of the card title.
+/// The bottom margin of the Card title.
 /// @group card
 $kendo-card-title-margin-bottom: map.get( $kendo-spacing, 3 ) !default;
-/// Font size of the card subtitle.
+/// The font size of the Card subtitle.
 /// @group card
 $kendo-card-subtitle-font-size: var( --kendo-font-size-sm, inherit ) !default;
-/// Font family of the card subtitle.
+/// The font family of the Card subtitle.
 /// @group card
 $kendo-card-subtitle-font-family: var( --kendo-font-family, inherit ) !default;
-/// Line height of the card subtitle.
+/// The line height of the Card subtitle.
 /// @group card
 $kendo-card-subtitle-line-height: var( --kendo-line-height, normal ) !default;
-/// Bottom margin of the card subtitle.
+/// The font weight of the Card subtitle.
```

Comment:
The variable name is $kendo-card-subtitle-margin-bottom -> please update its description